README.md

```diff
@@ -4,24 +4,27 @@ Various experiments using the python pymc3 library for Bayesian learning.
 
 ## Quick start
 
-Install the virtual environment `posteriorpymc3`:
+### Installing `uv`
 
-### Windows
+The Python project manager selected is `uv`. To install the project 
+dependencies, run the following command:
 
 ```bash
```

Comment:
It would be great to have more general installation instructions like the one using curl from the uv documentation.

Reply:
I add this change including the installation guide.